### PR TITLE
problem: no support for disconnect and hello msg socket options

### PR DESCRIPTION
**Pull request by somdoron**

### CODE REVIEW

File: api/zsock_option.api [hunk 1]
```diff
@@ -16,6 +16,18 @@
     <argument name = "only first subscribe" type = "integer" />
 </method>
 
+<method name = "set hello msg" polymorphic = "1">
```
Comment (bluca) [inline api/zsock_option.api:19]: these both need to be draft

Comment (bluca) [reply api/zsock_option.api:19]: actually never mind, it's not manually added

**Comment by bluca** [comment]
you need to regen, for the bindings

**Comment by somdoron** [comment]
regenerated

**Pull request merged**